devimint/src/gatewayd.rs

```diff
@@ -235,9 +235,14 @@ impl Gatewayd {
         let federation_id = fed.calculate_federation_id();
         let invite_code = fed.invite_code()?;
         info!("Recovering {federation_id}...");
-        cmd!(self, "connect-fed", invite_code.clone(), "--recover=true")
-            .run()
-            .await?;
+        poll("gateway recover-fed", || async {
```

Comment:
Nit: `connect-fed --recover` since `gateway recover-fed` isn't a real command

Reply:
its not meant to be a command, its just a string that indicates what went wrong